server.js

```diff
@@ -415,8 +419,35 @@ function prStatus (pr, delta, cb) {
                                         pr.contribStatus[username] = "ok";
                                         return cb(null, "ok");
                                     } else {
-                                        pr.contribStatus[username] = "not in group";
-                                        return cb(null, "not in group");
+                                        // we assume that all groups are of the same type
+                                        store.getGroup(repoGroups[0], function(err, group) {
+                                            if (err) return cb(err);
+                                            if (!group) return cb("Unknown group: " + repoGroups[0]);
+                                            if (group.groupType === 'WG') {
+                                              log.info("Looking up for non-participant licensing contribution");
+                                              gh.getRepo(pr, function(err, res) {
```

Comment:
The repo id appears in the [pull request output from the github API](https://developer.github.com/v3/pulls/#response-1) - you could save a request to `getRepo` here